test/inactive-blog-filter.test.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const inactiveFilter = require('../src/inactive-blog-filter');
+
+const redlist = JSON.parse(fs.readFileSync('feeds-redlist.json', 'utf8'));
```

Comment:
Change this to `const redlist = require('feeds-redlist.json');`

Reply:
Thank you for the review!
(Resolved by b8019ac.)